data.md

```diff
@@ -500,9 +500,9 @@ The local memory of execution is a byte-array (instead of a word-array).
     syntax WordStack ::= #asByteStack ( Int )             [function]
                        | #asByteStack ( Int , WordStack ) [function, klabel(#asByteStackAux), smtlib(asByteStack)]
  // --------------------------------------------------------------------------------------------------------------
-    rule #asByteStack( W ) => #asByteStack( W , .WordStack )                                        [concrete]
-    rule #asByteStack( 0 , WS ) => WS                                                            // [concrete]
-    rule #asByteStack( W , WS ) => #asByteStack( W /Int 256 , W modInt 256 : WS ) requires W =/=K 0 [concrete]
+    rule #asByteStack( W ) => #asByteStack( W , .WordStack )                                           [concrete]  
+    rule #asByteStack( W , WS ) => WS                                             requires W ==Int 0   // [concrete]
+    rule #asByteStack( W , WS ) => #asByteStack( W /Int 256 , W modInt 256 : WS ) requires W =/=K 0    [concrete]
 ```
```

Comment:
Here it's safe to use `0` directly for matching, because it's an internal function and the first argument is used just as a counter.